server/shared/request/mw.js

```diff
@@ -1,11 +1,53 @@
 'use strict';
 
 const rateLimit = require('express-rate-limit');
+const Tapster = require('@extensionengine/tapster');
+const { provider, ...options } = require('../../../config/server').store;
 
 const DEFAULT_WINDOW_MS = 15 * 60 * 1000; // every 15 minutes
 
-function requestLimiter({ max = 10, windowMs = DEFAULT_WINDOW_MS, ...opts } = {}) {
-  return rateLimit({ max, windowMs, ...opts });
+// Store must be implemented using the following interface:
+// https://github.com/nfriedly/express-rate-limit/blob/master/README.md#store
+class Store {
+  constructor() {
+    this.cache = new Tapster({
+      ...options[provider],
+      store: provider,
+      namespace: 'request-limiter'
+    });
+  }
+
+  async incr(key, cb) {
+    const initialState = { hits: 0 };
+    const record = await this.cache.has(key)
+      ? await this.cache.get(key)
+      : initialState;
+    const hits = record.hits + 1;
+    await this.cache.set(key, { ...record, hits });
+    cb(null, hits);
+  }
```

Comment:
Consider if it's worth it to make this and `decrement` a bit more concise:
```suggestion
  async incr(key, cb) {
    const initialState = { hits: 0 };
    const { hits, ...record } = await this.cache.has(key)
      ? await this.cache.get(key)
      : initialState;
    await this.cache.set(key, { ...record, hits: hits + 1 });
    cb(null, hits);
  }
```